### PR TITLE
Packaging group minutes, 2018/4/4

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
@bencouturier thanks for your help with the notes. Things were still a bit scrappy doing a lot of taking and writing, but if you can @ktf can give a quick check it would be good. Thanks.

File: organization/_posts/2018-04-04-packaging.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- HSF rebranding - use the new name for the mailing list: `hsf-packaging-wg@googlegroups.com`. 
+
+
+Summary of Packaging discussion in Naples - Giulio
+==================================================
+
+- Need to now do a sensible test drive on the tools.
+
+- Test cases for the development environment probably need further expansion.
+    - Incremental builds within packages might need to be reviewed (e.g., modify ROOT and rebuild only a few libraries without recompiling every downstream piece). This is not guaranteed to work, but with *caveat emptor*, can be useful to be able to accelerate development.
```
Comment (ktf) [inline organization/_posts/2018-04-04-packaging.md]: I was actually thinking that the only case in which it does not work is when you do not type "make" in all the dependent packages. aliBuild actually does that for you, if you want, so in principle is not so bad, assuming the inner build system is sensible (e.g. configure / make or cmake) and can spot when dependencies are modified.

Comment (graeme-a-stewart) [reply organization/_posts/2018-04-04-packaging.md]: OK, that would work when you have a complete build area already. How would it work if you don't? Then you can LD_LIBRARY_PATH in your new component?

Comment (ktf) [reply organization/_posts/2018-04-04-packaging.md]: I think the requirement we have is that you have built everything at least once. Also by well behaved I mean build systems which do not need LD_LIBRARY_PATH or any other environment in order to build.

Comment (ktf) [reply organization/_posts/2018-04-04-packaging.md]: That said, aliBuild would actually perform a full build in the case you mention.

**Pull request merged**